Makefile

```diff
@@ -41,25 +41,16 @@ DOC_FILES := \
 FIGURE_FILES := \
 	img/media-types.png
 
-TOOLS := esc gitvalidation 
+TOOLS := gitvalidation 
 
 default: check-license lint test
 
-help:
-	@echo "Usage: make <target>"
-	@echo
-	@echo " * 'docs' - produce document in the $(OUTPUT_DIRNAME) directory"
-	@echo " * 'fmt' - format the json with indentation"
-	@echo " * 'validate-examples' - validate the examples in the specification markdown files"
-	@echo " * 'check-license' - check license headers in source files"
-	@echo " * 'lint' - Execute the source code linter"
-	@echo " * 'test' - Execute the unit tests"
-	@echo " * 'img/*.png' - Generate PNG from dot file"
-
-fmt:
+.PHONY: fmt
+fmt: ## format the json with indentation
```

Comment:
meh. I don't _love_ having comments on the same line ...

Reply:
I've got the same hesitation, and would really like it if they could go on the previous line. But the awk command everyone copies for `make help` depends on that syntax.